src/lib/jsonParseV2/parseContextAttribute.cpp

```diff
@@ -121,11 +121,54 @@ static std::string parseContextAttributeObject(const Value& start, ContextAttrib
         return r;
       }
     }
-    else  // ERROR
+    else if (keyValues == false) // ERROR
     {
       LM_W(("Bad Input (unrecognized property for ContextAttribute - '%s')", name.c_str()));
       return "unrecognized property for context attribute";
     }
+    else  // Anything captured here is a metadata, and options=keyValues has been set
```

Comment:
Not sure of understand the comment "Anything captured here is a metadata, and options=keyValues has been set" as is keyValues is set, then no metadata should be captured... maybe I'm missing something. Could you elaborate, please?